pulp/pulp.py

```diff
@@ -144,28 +144,28 @@
 try:  # allow Python 2/3 compatibility
     maketrans = str.maketrans
 except AttributeError:
-    from string import maketrans
```

Comment:
Why not just change to be: `from string import maketrans`, Python 2 is not supported